### DISCUSSION TITLE
Docs: Improve scripts

### DISCUSSION THREAD

**Discussion by Jay-Karia** [vercel/next.js#69937] [Ideas]
### What is the documentation issue?

In some of the documentation pages (say https://nextjs.org/docs/pages/building-your-application/testing/jest#manual-setup)
the scripts are not separated from different package managers making the copy button useless as it copies all the commands with different package mangers.

At the same page (https://nextjs.org/docs/pages/building-your-application/testing/jest#quickstart) the command used is only written in `npx`, rather it should be in yarn and pnpm

### Is there any context that might help us understand?

Separate scripts for different package managers.
Add more package managers to some of the scripts

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/pages/building-your-application/testing/jest

**Comment by samcx** [comment]
@Jay-Karia Thank you for the feedback! This has been mentioned before, we have just not been able to get around to it.

Since this basically a feature request and not something incorrect in the docs, I have converted it to a discussion.
